styles.css

```diff
@@ -190,11 +185,21 @@ body {
 }
 
 .techs-container {
+  display: flex;
+  column-gap: 12px;
+  justify-content: flex-start;
+  list-style: none;
+  padding-left: 0;
+  margin: 20px 0 32px 0;
+}
+
+.techs-container-skills {
   display: flex;
   column-gap: 12px;
   justify-content: center;
```

Comment:
- [ ] As suggested by the previous reviewer you could use `justify-content: flex-start` or just remove this property in order to match the Figma design because your portfolio has these skills centered and these should be aligned to the left :smiley: :raised_hands: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144669091-41bfb85f-e98c-46f2-9431-38bae0b987b9.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144669125-02931187-6f9b-4d8c-84fb-f34e8a4fc2c7.png)